SeoPresentation.php

```diff
@@ -136,14 +136,9 @@ public function addExtractor(ExtractorInterface $extractor, $priority = 0)
     }
 
     /**
-     * Gets the SeoMetadata based on the content that contains the content.
-     *
-     * @param object $content
-     *
-     * @throws Exception\InvalidArgumentException
-     * @return SeoMetadata
+     * {@inheritDoc}
      */
-    private function getSeoMetadata($content)
+    public function getSeoMetadata($content)
```

Comment:
i was assuming that this is invalid, but actually its not:
http://3v4l.org/ZqIEm#v500 (maybe some stale php4 memory)

ok, lets document this BC break when extending the class.